en/components/bulletgraph.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Bullet Graph Component - Native Angular | Ignite UI for Angular 
+_description: The Bullet Graph Component in Ignite UI for Angular allows for a linear and concise view of measures compared against a scale.  
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Grid, Angular Data Grid, Angular Grid Control, Angular Grid Component, Angular data grid Bullet graph component example, Angular bullet graph 
+---
+## Bullet Graph
+
+The Bullet Graph Component in Ignite UI for Angular allows for a linear and concise view of measures compared against a scale.  
+
+### Demo
+
+<div class="sample-container" style="height: 550px">
+    <iframe id="bullet-graph-sample-iframe" src='{environment:demosBaseUrl}/bullet-graph-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn"   data-iframe-id="bullet-graph-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">View on StackBlitz
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+The bullet graph control provides you with the ability to create attractive data presentations, replacing meters and gauges that are used on dashboards with simple yet straightforward and clear bar charts. A bullet graph is one of the most effective and efficient ways to present progress towards goals, good/better/best ranges, or compare multiple measurements in as little horizontal or vertical space as possible.
+
+The features of the bullet graph include configurable orientation and direction, configurable visual elements, and more. The control has also a built-in support for animated transitions.
+
+### Dependencies
+The bullet graph is exported as an `NgModule`, you need to import the _IgxBulletGraphModule_ inside your `AppModule`:
+
+```typescript
+// app.module.ts
+import { IgxBulletGraphModule } from '@infragistics/ignite-ui-xplatform-for-angular/Debug/ES5/igx-bulletgraph-module';
+
+@NgModule({
+    imports: [
+        ...
+        IgxBulletGraphModule,
+        ...
+    ]
+})
+export class AppModule {}
+```
+
+<div class="divider--half"></div>
+
+### Usage
+
+The following code walks through creating a bullet graph control, and configuring a performance bar, comparative measure marker, and three comparative ranges on the scale.
+
+
+```html
+ <igx-bullet-graph height="100"
+                   width="300"
+                   minimumValue="5"
+                   maximumValue="55"
+                   value = "35"
+                   targetValue = "43">
+    <igx-bullet-graph.ranges>
```

Comment:
please remove `<igx-bullet-graph.ranges>` line because it will throw errors! It is needed only in XAML